Create an admin to the DB. Add --redirect to the certbot command

tzachi-dar opened the pull request:
Add --redirect to the certbot command. This will remove one question from the installation script.

tzachi-dar commented:
Also added a script that will move an existing installation to use freedns (no need to update mails).

jamorham commented:
It looks like your script attempts to every minute check the public ip using a remote web service. This is a bad idea. The IP address will only change at most on-boot. Unless we need to periodically poll freedns then we should do it just on boot but also is there no way to automatically set the ip based on the one they receive the request from? (that way we don't have to use another 3rd party site)

If we do need to do periodic polling and if out of the box cron doesn't work then instead look at systemd timers. https://wiki.archlinux.org/title/systemd/Timers and if possible build in some randomness.

tzachi-dar commented:
The one minute is something that was left from the development phase.
Free dns don't need us to supply any IP, but they don't want us to call them without a reason. This is why I had the call to checkip.dyndns.org. (assuming that we were calling them periodically). 
If you are sure that our IP only changes at boot time, then we can make the call one time at boot (and probably 5 minutes later just to make sure) and we should be fine.
In that case, is this OK that I keep using cron jobs (for running after the reboot only)? They seems to be easier to config in this cases.

jamorham commented:
The IP is tied to the instance, it typically wont change unless the instance is destroyed. I think you can just update the freedns on boot and easily chain it to the existing startup script.

tzachi-dar commented:
OK, I have changed the script to not use crontab. Instead it adds one command to the /etc/rc.local script.

Apart from that, I have also added the script GetHerokuVars.sh. This script copies all the variables from heroku to a local file, to make it easier for more complex nightscout installations.
